db/migrate/20221019160731_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] it would be better if you change `:user `to `author` as asked in the requirements and also you didn't specify the users' table so it doesn't know which table are you talking about cause there is no authors table 
please try to correct that by specifying a table like that ` foreign_key: { to_table: 'users' }`